.gitignore

```diff
@@ -1,6 +1,6 @@
 # This file is part of the Stooa codebase.
 #
-# (c) 2020 - present Runroom SL
+# (c) 2021 - present Runroom SL
```

Comment:
Changes on the license should be reverted since 2020 was the year we started working on Stooa.

Reply:
I see. Please just ignore this suggestion, then!